conf/iridanext.config

```diff
@@ -14,7 +14,6 @@ iridanext {
                 "**/arborator/*_profile.tsv",
                 "**/arborator/*_loci.summary.tsv",
                 "**/arborator/*_tree.nwk",
-                "**/arborator/*_matrix.pq",
```

Comment:
`matrix.pq` file is no longer produced as output by arborator, so removing inclusion here.